src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -46,4 +46,18 @@
   <Import Project="$(CoreCLRBuildIntegrationDir)Microsoft.DotNet.ILCompiler.targets"
           Condition="'$(NativeAotSupported)' == 'true' and '$(RunningPublish)' == 'true'" />
 
+  <Target Name="RewriteRuntimePackDir"
```

Comment:
Maybe add a TODO to consolidate this logic with targetingpack.targets and/or create a tracking issue?

Reply:
Yeah, I'll file a tracking issue.